### DISCUSSION TITLE
keyball61でジャンパのショートに失敗しランドが剥がれてしまいました。リカバリできますでしょうか。

### DISCUSSION THREAD

**Discussion by tadashi1127** [Yowkees/keyball#435] [Q&A]
お世話になります。
keyball61を購入させていただいて、現在組み立て中なのですが、
ジャンパのショートに失敗し、ランドが剥がれてしまいショートさせることができない状態になっていまいました。(2か所)
ここからリカバリする方法がありましたら、ご教授お願いします。

![IMG_20231203_195219](https://github.com/Yowkees/keyball/assets/152780542/9aed5c29-f0cb-4c89-a581-b1e25e103c1c)

お忙しいところ恐縮ですが、よろしくお願いします。

**Comment by Ryo624** [comment]
ご連絡ありがとうございます。

ジャンパ部分のランド剥がれについて了解しました。

添付いたしました写真の、
ランド剥がれ１は青点どうし、ランド剥がれ２は黄点どうしをエナメル線かゴム被覆線でつないでいただきますとジャンパをショートしたことになりますので、お手数ですがご対応のほどよろしくお願いいたします。

また、こちらの作業はTRSジャック、L字コンスルーのはんだ付けを終えた後に行ってください。

![287512963-9aed5c29-f0cb-4c89-a581-b1e25e103c1c](https://github.com/Yowkees/keyball/assets/143698349/2680e621-97fe-45a1-a6b7-0dfd47d44010)

**Reply by tadashi1127** [reply]
@Ryo624 
迅速な回答ありがとうございます！
案内していただいた箇所をエナメル線で繋いで対処しようと思います。

対処後にまた報告させていただきます。
組み立ての続きを行えるのが週末になる見込みですので、報告まで少し間が開くと思いますが、
よろしくお願いします。

**Comment by tadashi1127** [comment]
無事に完成させることができました。
ありがとうございました。
本件クローズします。
